### ISSUE TITLE
DOTA2 :7.28  clarity:2.5 hotfix  BUG

### ISSUE THREAD

**Issue by yangex** [skadistats/clarity#248]
match_id:5746444621
error:
```
skadistats.clarity.ClarityException: Invalid memory access: Tried to access array of length 366 at offset 376
	at skadistats.clarity.decoder.bitstream.UnsafeBitStreamBase.accessFailed(UnsafeBitStreamBase.java:53) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.bitstream.UnsafeBitStreamBase.checkAccessRelative(UnsafeBitStreamBase.java:47) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.bitstream.UnsafeBitStream64.readUBitInt(UnsafeBitStream64.java:30) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.bitstream.BitStream.readVarU(BitStream.java:133) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.bitstream.BitStream.readVarUInt(BitStream.java:156) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.unpacker.IntVarUnsignedUnpacker.unpack(IntVarUnsignedUnpacker.java:9) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.unpacker.IntVarUnsignedUnpacker.unpack(IntVarUnsignedUnpacker.java:5) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.s2.S2FieldReader.readFields(S2FieldReader.java:77) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.decoder.s2.S2FieldReader.readFields(S2FieldReader.java:14) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.entities.Entities.getBaseline(Entities.java:486) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.entities.Entities.processEntityCreate(Entities.java:372) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.entities.Entities.onPacketEntities(Entities.java:317) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.Event.raise(Event.java:41) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.reader.InputSourceProcessor.processEmbedded(InputSourceProcessor.java:247) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.Event.raise(Event.java:41) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.reader.InputSourceProcessor.processSource(InputSourceProcessor.java:147) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.event.Event.raise(Event.java:41) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.runner.AbstractFileRunner.initAndRunWith(AbstractFileRunner.java:39) ~[clarity-2.5-SNAPSHOT.jar:na]
	at skadistats.clarity.processor.runner.ControllableRunner.lambda$runWith$0(ControllableRunner.java:238) ~[clarity-2.5-SNAPSHOT.jar:na]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_201]
```
code:

```
EndProcessor endProcessor = new EndProcessor();
ControllableRunner runner = new ControllableRunner(new MappedFileSource(filePath)).runWith(endProcessor);
runner.seek(runner.getLastTick());
```

when i use ControllableRunner.seek(ControllableRunner.getLastTick()), Please help me！

**Comment by STRATZ-Ken** [comment]
Probably should talk in #247

**Comment by spheenik** [comment]
Hey @yangex , I think I have a fix, I'll close this and move all 7.28 conversation to #247
